appservice/src/main/java/nu/yona/server/messaging/rest/MessageController.java

```diff
@@ -131,9 +130,15 @@ private GoalIdMapping createGoalIdMapping(UserDto user)
 		return GoalIdMapping.createInstance(user);
 	}
 
-	public MessageDto toMessageResource(GoalIdMapping goalIdMapping, MessageDto message)
+	public HttpEntity<MessageDto> createOkResponse(GoalIdMapping goalIdMapping, MessageDto message)
 	{
-		return new MessageResourceAssembler(curieProvider, goalIdMapping, this).toResource(message);
+		return createOkResponse(message, createResourceAssembler(goalIdMapping));
+	}
+
+	public HttpEntity<PagedResources<MessageDto>> createOkResponse(PagedResourcesAssembler<MessageDto> pagedResourcesAssembler,
```

Comment:
As order of parameters I would expect `messages` first as in `ControllerBase`.

Reply:
Will do.